chart/gatekeeper-operator/values.yaml

```diff
@@ -0,0 +1,10 @@
+# Default values for gatekeeper-operator.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+replicas: 1
+auditInterval: 60
+constraintViolationsLimit: 20
+image: quay.io/open-policy-agent/gatekeeper:v3.0.4-beta.2
```

Comment:
can we consider something like this for the image so that we only need to update the tag when we have a release.

```yaml
image:
  repository: quay.io/open-policy-agent/gatekeeper
  tag: v3.0.4-beta.2
  pullPolicy: IfNotPresent
```

Reply:
Sure, good idea